### PR TITLE
 update fedora patch number in spec file

### PR DESCRIPTION
**Pull request by anandaravuri**
- Update fedora patch number 2 in spec and create rpm

Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: opae.spec [hunk 1]
```diff
@@ -9,7 +9,7 @@ Group:          Development/Libraries
 Vendor:         Intel Corporation
 Requires:       uuid, json-c, python3
 URL:            https://github.com/OPAE/%{name}-sdk
-Source0:        https://github.com/OPAE/opae-sdk/releases/download/%{version}-%{Release}/%{name}-%{version}.tar.gz
```
Comment (r-rojo) [inline opae.spec:12]: Why can't we use %{Release}? It's set to 2 (or dist) in line 4.

Comment (anandaravuri) [reply opae.spec:12]: https://bugzilla.redhat.com/show_bug.cgi?id=1806017 

https://github.com/trixirt/opae-rpm/blob/opae-1.4.0-5/opae.spec

https://fedoraproject.org/wiki/Packaging:DistTag


Tom was updated spec file like below,  followed his  pattern

Version:        1.4.0
Release:        5%{?dist}
Source0:        https://github.com/OPAE/opae-sdk/releases/download/%{version}-1/%{name}-%{version}-1.tar.gz

Comment (tswhison) [reply opae.spec:12]: This seems like a step backward in terms of maintainability.  I don't see why we would need to hard-code things and require that so many changes be made each time.

Comment (r-rojo) [reply opae.spec:12]: Yes, hardcoding for a release would work to but this doesn't mean we can't use a variable.
We are already setting it in line 4. Using the variable should work just as well as using the literal value except that now we only have to change it in once place if/when we have to change it.

Comment (anandaravuri) [reply opae.spec:12]: set %{Release}? to 2 in spec file

**Review by michelleyho — APPROVED** [review]

File: opae.spec [hunk 2]
```diff
@@ -1,15 +1,15 @@
 Summary:        Open Programmable Acceleration Engine (OPAE) SDK
 Name:           opae
 Version:        2.0.0
-Release:        2%{?dist}
+Release:        2
```
[the remark this answers is not in the document]
Comment (anandaravuri) [reply opae.spec]: fixed  rojo comments

**Review by r-rojo — APPROVED** [review]

**Pull request merged**